src/Cassandra/RecyclableMemoryStreamManager.cs

```diff
@@ -290,9 +289,8 @@ internal byte[] GetBlock()
         /// will be at least the requiredSize and always be a multiple of largeBufferMultiple.
         /// </summary>
         /// <param name="requiredSize">The minimum length of the buffer</param>
-        /// <param name="tag">The tag of the stream returning this buffer, for logging if necessary.</param>
         /// <returns>A buffer of at least the required size.</returns>
-        internal byte[] GetLargeBuffer(int requiredSize, string tag)
+        internal byte[] GetLargeBuffer(int requiredSize)
```

Comment:
FYI: Removed unused parameter tag

Reply:
I know that this parameter is not used atm but it could be very useful for trace/verbose logging as the XML comment states so I'd rather keep them than removing them now and readding them later. There's a couple other places with this same parameter and it's the same scenario.